### PR TITLE
New package: xfce4-calculator-plugin 0.7.1

### PR DESCRIPTION
**Pull request by IT-DEVS-Group**
- I tested the changes in this PR: YES
- I built this PR locally for my native architecture, (x86_64 musl, 86_64 glib)

### CODE REVIEW

File: srcpkgs/xfce4-calculator-plugin/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'xfce4-calculator-plugin'
+pkgname=xfce4-calculator-plugin
+version=0.7.1
+revision=1
+build_style=gnu-configure
+configure_args="--with-locales-dir=/usr/share/locale"
+hostmakedepends="pkg-config intltool"
+makedepends="xfce4-panel-devel"
+short_desc="Battery monitor plugin for the Xfce panel"
```
Comment (Duncaen) [inline srcpkgs/xfce4-calculator-plugin/template:9]: The description doesn't look like it matches the name.

Comment (IT-DEVS-Group) [reply srcpkgs/xfce4-calculator-plugin/template]: > The description doesn't look like it matches the name.

Fixed

Comment (IT-DEVS-Group) [reply srcpkgs/xfce4-calculator-plugin/template]: ha

File: common/environment/setup/misc.sh [hunk 2]
```diff
@@ -18,5 +18,6 @@ GNU_SITE="https://ftp.gnu.org/gnu"
 FREEDESKTOP_SITE="https://freedesktop.org/software"
 KDE_SITE="https://download.kde.org/stable"
 VIDEOLAN_SITE="https://download.videolan.org/pub/videolan"
+XFCE_SITE="https://archive.xfce.org"
```
Comment (paper42) [inline common/environment/setup/misc.sh:21]: Drop the commit with this change, we agreed to not add this variable in another PR.

Comment (IT-DEVS-Group) [reply common/environment/setup/misc.sh]: done

Comment (paper42) [reply common/environment/setup/misc.sh]: You also can't use this variable in the package when it's not defined.

**Comment by IT-DEVS-Group** [comment]
ha-ha. mess. ok. done and bye

File: srcpkgs/xfce4-calculator-plugin/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+version=0.7.1
+revision=1
+build_style=gnu-configure
+configure_args="--with-locales-dir=/usr/share/locale"
+hostmakedepends="pkg-config intltool"
+makedepends="xfce4-panel-devel"
+short_desc="Calculator plugin for the Xfce4 panel"
+license="GPL-2.0-or-later, LGPL-2.0-or-later"
+homepage="https://docs.xfce.org/panel-plugins/xfce4-calculator-plugin"
+changelog="https://gitlab.xfce.org/panel-plugins/xfce4-calculator-plugin/-/blob/master/NEWS"
```
Comment (paper42) [inline srcpkgs/xfce4-calculator-plugin/template:12]: link to a raw text file

Comment (IT-DEVS-Group) [reply srcpkgs/xfce4-calculator-plugin/template:12]: and so it goes. anyway, when xfce changes the server name, with this speed of maintainers and this approach, the packages will not be available for a long time. then they'll fix it.

Comment (IT-DEVS-Group) [reply srcpkgs/xfce4-calculator-plugin/template:12]: raw is prefered but not required

Comment (paper42) [reply srcpkgs/xfce4-calculator-plugin/template:12]: > and so it goes. anyway, when xfce changes the server name, with this speed of maintainers and this approach, the packages will not be available for a long time. then they'll fix it.

I have no idea what you are talking about.

raw changelog is prefered which means packages should use it if it's available, please change it. Also, you will have to fix CI before this is merged.

**Comment by paper42** [comment]
Why did you close this? This PR would be merged if you fixed CI and adressed my review.

**Pull request closed without merging**